tests/package.json

```diff
@@ -29,11 +29,12 @@
     }
   },
   "scripts": {
-    "test-with-logs": "mocha --printlogs -r ts-node/register 'tests/**/*.ts'",
-    "test": "mocha --parallel -r ts-node/register -r fixtures/contract-fixtures.ts 'tests/**/*.ts'",
+    "test-with-logs": "mocha --printlogs -r ts-node/register 'tests/**/test-*.ts'",
+    "pre-build-contracts": "TS_NODE_TRANSPILE_ONLY=true ts-node ./pre-build-contracts.ts && npx prettier -w ./contracts/compiled/*.json",
```

Comment:
has this been added to the CI? Or is it supposed to be ran by anyone who adds a contract?